src/pages/roadmap/StatusFeedbackList.vue

```diff
@@ -13,7 +13,7 @@
           <!-- list of items dependly on status name -->
           <h3></h3>
           <!-- status name here, use v-for to attach the list, h3 name and description below dependly on the status name -->
-          <p>{{ description }}</p>
+          <p></p>
```

Comment:
is it done on purpose?

Reply:
this is another page, only the basic skeleton has been done at the beginning and for sure this interpolation would be changed. I had a bug during the deployment and it seems to be connected with that page - I commented this code to fix the bug for deployment and you see that something has been changed. Now I learned that better is to verify the code by npm preview.